Data/sync/Sync.swift

```diff
@@ -179,6 +182,9 @@ public class Sync: JSInjector {
         fetchTimer = nil
         
         KeychainWrapper.standard.removeObject(forKey: Preferences.Sync.seedName.key)
+        // Edge case(#846). If a user stays on any of Sync screens while another device removes this device
```

Comment:
Is this actually an edge case still? Maybe just remove the `Edge case(#846). ` as this technically isn't an edge case, but rather a solution to the above issue?

Reply:
You are right, I called it that way because it's rather unlikely that a user will be in one of the sync screens when his device gets deleted from another device.